### ISSUE TITLE
Paperclip not refreshing missing thumbnail styles for models with attachments, defined in an engine

### ISSUE THREAD

**Issue by mitio** [thoughtbot/paperclip#1138]
I had an issue with the `paperclip:refresh:missing_styles` Rake task not regenerating the missing styles for one of my models. The model was defined in a Rails engine.

I guess a quick solution would be to replace the code on [this line](https://github.com/thoughtbot/paperclip/blob/master/lib/tasks/paperclip.rake#L64) with something along the lines of:

    # Force loading all model classes to never miss any has_attached_file declaration:
    Rails.application.eager_load!

This will ensure proper preloading of all models in production, even ones defined in engines.

Some more info on the subject can be found in this [SO answer](http://stackoverflow.com/a/4308167/75715).

Any thoughts?

**Comment by maclover7** [comment]
Hi @mitio ! Is this still an issue for you in Paperclip; I know this issue is from approximately 2 years ago. If it is still an issue, can you please provide the code that's causing you the error? Thanks!

**Comment by mitio** [comment]
@maclover7 I think this issue has been solved in 11734e2da8de19e3c76591193b1a64e16c128607. The solution originally targeted #1520, but should fix this issue as well. It's the same thing I suggested in my original report.

Thanks for reminding me to follow back on this.

**Comment by wmakley** [comment]
I am having the same issue with Rails 5.0.2 and paperclip 5.1.

I have the following model defined in an engine:

```ruby
module Callouts
  class Callout < ActiveRecord::Base
    has_attached_file :file, styles: {small: '200x100#', large: '233x317#'}
  end
end
```

```sh
rake paperclip:refresh:missing_styles
```

Does nothing!
